transition-scripts/values/eb75ad/prod/values-gold-eb75ad-prod-active.yaml

```diff
@@ -0,0 +1,52 @@
+replicaCount: 3
+
+additionalServerOptions: >-
+  -Dkeycloak.profile.feature.impersonation=disabled
+  -Djboss.persistent.log.dir=/var/log/eap
+  -Djgroups.dns.query=sso-keycloak-ping.eb75ad-prod.svc.cluster.local
+
+extraEnvs:
+  - name: KC_HOSTNAME_URL
+    value: https://loginproxy.gov.bc.ca/auth/
+  - name: KC_HOSTNAME_ADMIN_URL
+    value: https://loginproxy.gov.bc.ca/auth/
+  - name: QUARKUS_LOG_FILE_ROTATION_MAX_BACKUP_INDEX
+    value: "42"
+  - name: QUARKUS_LOG_FILE_ROTATION_MAX_FILE_SIZE
+    value: 1M
```

Comment:
I feel this one is too low value for production instance?? @thegentlemanphysicist

Reply:
I remember talking about this param with Jon L and keeping it at 200 M for prod ones especially.

Reply:
The update is now merged in